domain/schema/controller/triggers/model-triggers.gen.go

```diff
@@ -8,7 +8,6 @@ import (
 	"github.com/juju/juju/core/database/schema"
 )
 
-
```

Comment:
Not sure I understand this change ?

Reply:
`go generate` made this change - somehow an inconsistent copy go landed